Formula/s/serialosc.rb

```diff
@@ -35,7 +35,7 @@ class Serialosc < Formula
   end
 
   def install
-    system "python3", "./waf", "configure", "--enable-system-libuv", "--prefix=#{prefix}"
+    system "python3", "./waf", "configure", "--prefix=#{prefix}"
```

Comment:
This configure option was [removed in 1.4.4](https://github.com/monome/serialosc/releases/tag/v1.4.4)